#geometry_msgs/Vector3 linear_acceleration
#geometry_msgs/Vector3 angular_velocity
float32[6] data